src/WebJobs.Script/Workers/ProcessManagement/IProcessRegistry.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Workers
 {
-    internal interface IProcessRegistry
+    public interface IProcessRegistry
```

Comment:
leave these as internal. If Grpc project needs them, add InternalsVisible to flag

Reply:
Thanks, that's a good idea. I just updated the PR.